src/GuildProfile/locales/es-ES.json

```diff
@@ -0,0 +1,30 @@
+{
+    "GUILD_PROFILE": "Perfil del Servidor",
+    "GUILD_PROFILE_GUILD_INFO": "Información del Servidor",
+    "GUILD_PROFILE_FRIENDS_IN_GUILD": "Amigos",
+    "GUILD_PROFILE_BLOCKED_USERS_IN_GUILD": "Usuarios Bloqueados",
+    "GUILD_PROFILE_LOADING": "Cargando",
+    "GUILD_PROFILE_CLICK_TO_COPY_SERVER_ICON_URL": "Click para copiar la URL del ícono del servidor",
+    "GUILD_PROFILE_CREATED_AT": "Creado el",
```

Comment:
Shouldn't this be `Creado en` since it refers to a date?

Reply:
no, "Creado en el" >> "Creado el",it is like it is not to it isn't